src/main/asciidoc/project/general_procedures/upgrade.adoc

```diff
@@ -0,0 +1,4 @@
+To upgrade to the latest version of the {productName} software, replace all currently used {productName} jars with their latest versions.
+However, before replacing the jars, ensure that all pending transactions are either completed or recovered.
+It is essential that the Object Store is emptied before transitioning to a newer version of {productName}.
```

Comment:
I think it might be better to say "empty" rather than "emptied" (so as not to leave within scope an idea of just deleting it!)

Reply:
+1

Reply:
Thank you